### PR TITLE
FLA-686 Moved Filing Comments into a tabbed section

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

https://justice.gov.bc.ca/jira/browse/FLA-686

- Created a tabbed section on the Package Review screen (bootstrap for now - to be replaced with shared-components)
- Fixed bug when displaying submitted date (wasn't showing the time portion correctly)
- Updated tests to ensure owasp (html content is properly escaped from user data)
- Add new dev dependency `moment-timezone` to force github to run tests in the BC timezone (it was running in UTC, not UTC-8)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

yarn line
yarn test
yarn test -u

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by yousiefc** [comment]
Not sure if that's how it is in the wireframe but is there meant to be a second title inside the content?

**Comment by KDNeufeld** [comment]
Yeah, I had the same question, but that is how the wireframe has it.

**Review by yousiefc — COMMENTED** [review]
Looks good to me

File: src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import React, { useEffect, useState } from "react";
 import FileSaver from "file-saver";
 import moment from "moment";
 import PropTypes from "prop-types";
+import Tabs from "react-bootstrap/Tabs"; /* TODO: replace with shared-components */
+import Tab from "react-bootstrap/Tab"; /* TODO: replace with shared-components */
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js:8]: nice TODO

**Review by alexjoybc — APPROVED** [review]
Looks good thanks @KDNeufeld

**Pull request merged**